app/helpers/application_helper.rb

```diff
@@ -1,2 +1,128 @@
 module ApplicationHelper
+
+  def recreation_list
+    [
+      [' '],
+      ['Ski'],
+      ['Board'],
+      ['Both'],
+      ['Other']
+    ]
+  end
+
+  def trip_list
+    [
+      [' '],
+      ['One Way'],
+      ['Round Trip']
+    ]
+  end
+
+  def pass_list
+    [
+      [' '],
+      ['IKON'],
+      ['Epic'],
+      ['Other'],
+      ['No Preference'],
+      ['No Pass']
+    ]
+  end
+
+  def dates_list
+    [
+      [' '],
+      ['Monday'],
+      ['Tuesday'],
+      ['Wednesday'],
+      ['Thursday'],
+      ['Friday'],
+      ['Saturday'],
+      ['Sunday'],
+      ['Weekend Fri-Sun'],
+      ['Any Day']
+    ]
+  end
+
+  def days_list
+    [
+      [' '],
+      ['01'],
+      ['02'],
+      ['03'],
+      ['04'],
+      ['05'],
+      ['06'],
+      ['07'],
+      ['08'],
+      ['09'],
+      ['10'],
+      ['11'],
+      ['12'],
+      ['13'],
+      ['14'],
+      ['15'],
+      ['16'],
+      ['17'],
+      ['18'],
+      ['19'],
+      ['20'],
+      ['21'],
+      ['22'],
+      ['23'],
+      ['24'],
+      ['25'],
+      ['26'],
+      ['27'],
+      ['28'],
+      ['29'],
+      ['30'],
+      ['31']
+    ]
+  end
+
+  def month_list
+    [
+      [' '],
+      ['Jan'],
+      ['Feb'],
+      ['Mar'],
+      ['Apr'],
+      ['May'],
+      ['Jun'],
+      ['Jul'],
+      ['Aug'],
+      ['Sep'],
+      ['Oct'],
+      ['Nov'],
+      ['Dec']
+    ]
+  end
+
+  def year_list
+    [
+      [' '],
+      ['2019'],
+      ['2020']
+    ]
+  end
+
+  def seats_list
+    [
+      [' '],
+      ['1'],
+      ['2'],
+      ['3'],
+      ['4'],
+      ['5']
+    ]
+  end
+
+  def destination_list
+    [
+      [' '],
+      ['Keystone']
+    ]
```

Comment:
This is what we were talking about earlier with Nathan building out the API to get the mountain locations list and then us putting those in here, correct?

Reply:
Correct.